examples/azure/main.tf

```diff
@@ -1,7 +1,29 @@
 provider "gardener" {
   profile              = "<my-gardener-project>"
   azure_secret_binding = "<my-gardener-azure-secret>"
-  kube_path            = "<my-gardener-service-account-kubeconfig>"
+  kube_path          = "${file("<my-gardener-service-account-kubeconfig>")}"
+  /*kube_path          =<<-EOT
+    kind: Config
+    clusters:
+      - cluster:
+          certificate-authority-data: >-
+            LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURQWpLZ0F3SUJBZ0lVWjRTUWRmUEVUdmRnRmVQTVh0dE41RnJVaDFJd0RRWUpLb1pJaHZjTkFRRUwKQlFBd0t6RXBNQ2NHQTFVRUF4TWdkbWx5ZEhWaGJDMW5ZWEprWlc0NlkyRTZhM1ZpWlMxaGNHbHpaWEoyWlhJdwpIaGNOTVRneE1qRTVNVEl5TVRBd1doY05Nak14TWpFNE1USXlNVEF3V2pBck1Ta3dKd1lEVlFRREV5QjJhWEowCmRXRnNMV2RoY21SbGJqcGpZVHByZFdKbExXRndhWE5sY25abGNqQ0NBU0l3RFFZSktvWklodmNOQVFFQkJRQUQKZ2dFUEFEQ0NBUW9DZ2dFQkFOWEkvZ0E3YWVpVkxLVGVCVEpxYk5hM3JrOHpaVUcrNXlnLy9WU1RlMlVtb21rZworMlNxY291Zis4S05FM0ZSL1hyZTR5cWM1S2dXWEdKUWdxMWl1a3RWdnVBRzFSRjZjamFMQlE0MWNZNHF6TGJ6Ci9TdFhIRVVMR3NhRENGaGRKZ0wzNXJjbHh0Zi9OcGRvczczUzhvQnV2djFPVk1NSDY3Y0toRkxOajFSZWdxcHMKN2ZhcXAyZXJsNVB1UlZiSjY5d29xK0E3YS9GUktKN3JBbUNrUzhaeHRaVHZ3Y2ZVUDNPbzl4bEdlQkh1VUlvUgowMW9HYlc2WGpjNDNTbytlcFRkUXBxZTNWNjhqeGgybkV4NXFheDZ4YjJJTC9VUXlpQlRpbEhFTkN2Tk1WZ2JqCnE4TUNCSjduUGhoZGkvRTJrZjRYY2FGRXZycHN0YzNQeTFXV1NVRUNBd0VBQWFObU1HUXdEZ1lEVlIwUEFRSC8KQkFRREFnRUdNQklHQTFVZEV3RUIvd1FJTUFZQkFmOENBUUl3SFFZRFZSME9CQllFRkxubkNJWVlwdHdhQ05SbwpXbGgrOWgwd2J0WlhNQjhHQTFVZEl3UVlNQmFBRkxubkNJWVlwdHdhQ05Sb1dsaCs5aDB3YnRaWE1BMEdDU3FHClNJYjNEUUVCQ3dVQUE0SUJBUUJZODhSWWFZb0NBVXlxSmk4VDFyeE5ENEJQeUYxZTlFemVBMGhXdUVjbzdFQmcKazdDU2NlamFyNVRUNDM2L1kvV1Mra0Z3bUF1L3I3OWtIMTB3MVA3UnI2dEFCZ1BJci91MExVako5NXc1eG5idQphTW9DdVB4SXY2SUdCNzVhZkhUVkxRbkNGRDBZS0NmRXVua2JZS09veWJWRGJwYUwyNDhWSnl6cURac0ZRU2V5ClJ1QkhoeTdtRzRHSlozaktwdExVU3JOMmowemV2U2E2Vmo1L2Vic2g0UDgvVXBteEpuMXRoRGZJNUcrdEZ0cTEKbmdVTkRsTjJWejNJdThBLzdSVlRxb1hGTzREcGJaUHFnR2FyZUVJWmNPN1JzYjBKc0xFZzFwWGpvVmo0bkNYSgphMmdadWdBVnFZOMVBROFp0bFRKbDFnUSt2bUljTlRtakFWCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0=

```

Comment:
Please delete any certificates or tokens, this could pose a security issue.

Reply:
fixed it